src/index.js

```diff
@@ -103,68 +109,62 @@ class Game extends React.Component {
   }
   
   render() {
-	const history = this.state.history;
-  const current = history[this.state.stepNumber];
-  const winner = calculateWinner(current.squares);
-  let Xcoloring = 1;
-	
-	const moves = history.map((step, move) => {
-      const desc = move ?
-        'Go to move #' + move + ' (' + (move % 2 ? 'X' : 'O') + ' at position ' + history[move].location + ')':
-        'Go to game start';
-      if (move % 2) {
-        return (
-          <li key={move}>
-            <button onClick={() => this.jumpTo(move)}>
-            {move === this.state.stepNumber ? <b className = "highlightedX" style = {{marginLeft:"0"}}>{desc}</b> : desc}
-            </button>
-          </li>
-        );
-      } else {
+    const boardSize = this.state.boardSize;
+    const history = this.state.history;
+    const current = history[this.state.stepNumber];
+    const winner = calculateWinner(current.squares, boardSize);
+    const draw = calculateIfDraw(current.squares, boardSize);
+    let Xcoloring = 1;
+    
+    const moves = history.map((step, move) => {
+        const desc = move ?
+          'Go to move #' + move + ' (' + (move % 2 ? 'X' : 'O') + ' at position ' + history[move].location + ')':
+          'Go to game start';
+        
         return (
           <li key={move}>
             <button onClick={() => this.jumpTo(move)}>
-            {move === this.state.stepNumber ? <b className = "highlightedO" style = {{marginLeft:"0"}}>{desc}</b> : desc}
+            {move === this.state.stepNumber ? <b className = {"highlighted" + (move % 2 ? "X" : "O")} style = {{marginLeft:"0"}}>{desc}</b> : desc}
             </button>
           </li>
         );
-      }
-      
     });
-	
+    
     let status;
     let status1;
-    if (winner === "draw") {
-		status = 'It\'s a draw!';
-	} else if (winner) {
-      status = 'Winner: ';
-      status1 = winner.player;
+    if (winner) {
+        status = 'Winner: ';
+        status1 = winner.player;
```

Comment:
You probably don't need this `status1`. You already have player in the winner variable. Further more, name is confusing (status1?).

Reply:
Fixed, thanks